### ISSUE TITLE
Alias 'From' name includes name and email address

### ISSUE THREAD

**Issue by danphilibin** [anonaddy/anonaddy#137]
Hi, I'm trying to figure out why all of my alias's names have a sanitized version of their email address in the full contact name:

![CleanShot 2021-03-14 at 14 15 57@2x](https://user-images.githubusercontent.com/180350/111084626-86872580-84d0-11eb-91e5-32ae0effc50d.png)

I'd prefer the contact's name was simply "Bookshop". This happens with all my aliases in Gmail and HEY, so I'm assuming that sender name formatting is coming from AnonAddy. (is [this](https://github.com/anonaddy/anonaddy/blob/master/app/Mail/ForwardEmail.php#L126) where that happens?)

Two questions:
1. Is there a reason this is the default behavior? 
2. Would you consider a setting to change the default From name format?

**Comment by willbrowningme** [comment]
Yes that is the correct location in the code where the "From:" header is set.

1. Yes that is default behaviour. The reason the @ symbol is replaced by " at " is because if you include an email address in the "From:" header of a message many email providers interpret it as an attempt to spoof a sender or as a phishing attempt which can result in the email landing in spam or being rejected completely.
2. Yes I will look into adding an option to choose how this is formatted.

**Comment by rossdargan** [comment]
Ideally I would like the from name part of the email just to match the original from email. It would be great to see this changed.

**Comment by xanoni** [comment]
> Ideally I would like the from name part of the email just to match the original from email. It would be great to see this changed.

You can set a custom "From" name in the AnonAddy settings ("Update From Name"). We probably don't want to just copy the "From" from your real email account, given it would de-anonymize you.

**Comment by rossdargan** [comment]
> > Ideally I would like the from name part of the email just to match the original from email. It would be great to see this changed.
> 
> You can set a custom "From" name in the AnonAddy settings ("Update From Name"). We probably don't want to just copy the "From" from your real email account, given it would de-anonymize you.

I mean the emails you recieve. The from name is changed from the senders name to the senders name plus the email address.

**Comment by xanoni** [comment]
@rossdargan so you want it to display just "Bookshop", without the email address? I guess that's something the AnonAddy team could implement.

**Comment by shumink** [comment]
I would love to see this as an option

**Comment by willbrowningme** [comment]
This will be added shortly, discussion for it is here - https://github.com/anonaddy/anonaddy/discussions/370.
